managed/services/supervisord/supervisord.go

```diff
@@ -675,6 +675,34 @@ stdout_logfile_backups = 3
 redirect_stderr = true
 {{end}}
 
+{{define "vmgateway"}}
+[program:vmgateway]
+priority = 7
+command =
+    /usr/sbin/vmgateway
+      -eula
+      -enable.auth
+      -envflag.enable
+      -envflag.prefix=VMG_
+      -http.disableResponseCompression
+      -write.url=http://127.0.0.1:9090/prometheus
+      -read.url=http://127.0.0.1:9090/prometheus
```

Comment:
Also it seems that we should check nginx configuration to be sure that VM is not accessible directly anymore

Reply:
Here's the related PR for this change https://github.com/percona/pmm-update/pull/252
As researched by Maxim, it shall be sufficient to pass `/prometheus/api/v1` through `vmgateway`.